study-builder/studies/osteo/study.conf

```diff
@@ -2417,7 +2418,8 @@
         ! (user.studies["CMI-OSTEO"].forms["CONSENT_ASSENT"].hasInstance()
         && user.studies["CMI-OSTEO"].forms["RELEASE_MINOR"].hasInstance()
         && user.studies["CMI-OSTEO"].forms["CONSENT_ASSENT"].isStatus("COMPLETE")
-        && user.studies["CMI-OSTEO"].forms["RELEASE_MINOR"].isStatus("COMPLETE"))
+        && user.studies["CMI-OSTEO"].forms["RELEASE_MINOR"].isStatus("COMPLETE")
+        && !user.studies["CMI-OSTEO"].forms["RELEASE_SELF"].hasInstance())
```

Comment:
needed to check if minor is aged-up and if so cancel pdf generation with minor template